.jshintrc

```diff
@@ -18,10 +18,13 @@
   "predef": [
     "exports", 
     "require",
-    "process"
+    "process",
+    "expect"
   ],
   "esnext": true,
   "shadow": false,
   "supernew": false,
-  "strict": false
+  "strict": false,
+  "mocha": true,
+  "eqeqeq": false
```

Comment:
I noticed you were using double equals and JShint was screaming at me. eqeq fixes it